Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1646,8 +1647,12 @@ void CGContextFillRects(CGContextRef context, const CGRect* rects, size_t count)
 */
 void CGContextDrawPath(CGContextRef context, CGPathDrawingMode mode) {
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
-    context->ClearPath();
+    if (context->HasPath()) {
+        ID2D1Geometry* pGeometry;
```

Comment:
Unless you're at a module boundary, or a boundary that makes sense (like CGContext <-> CGPath), and even then only in function parameters.